pkg/audit/manager.go

```diff
@@ -213,6 +212,18 @@ func (am *Manager) auditResources(ctx context.Context) ([]*constraintTypes.Resul
 	var responses []*constraintTypes.Result
 	var errs opa.Errors
 
+	type labels map[string]string
+	// nsCache is used for caching namespaces and their labels
+	nsCache := make(map[string]labels)
```

Comment:
This should just cache the entire namespace object. This will future proof things in case more parts of the namespace object becomes relevant.